powdr_cli/Cargo.toml

```diff
@@ -12,6 +12,8 @@ parser = { path = "../parser" }
 riscv = { path = "../riscv" }
 number = { path = "../number" }
 halo2 = { path = "../halo2" }
+backend = { path = "../backend" }
+pil_analyzer = { path = "../pil_analyzer" }
```

Comment:
Ideally `cli` would not depend directly on `pil_analyzer` and instead go through `compiler`

Reply:
It's not a big deal now though I reckon.

Reply:
Probably better to fix it now